tests/backends/models.py

```diff
@@ -113,3 +113,18 @@ def __str__(self):
 
 class RawData(models.Model):
     raw_data = models.BinaryField()
+
+# We need these models only for Oracle DB tests
+if connection.vendor == 'oracle':
+    class Student(models.Model):
+        name = models.CharField(max_length=30)
+
+        class Meta:
+            db_table = '"test_%s"."DJANGO_STUDENT_TABLE"' % (connection.settings_dict['USER'])
```

Comment:
@charettes I am not quite sure if I properly understand you but I guess this one is using exactly test connection because it's using default test connection user setting